### ISSUE TITLE
Problem in formatISODuration documentation

### ISSUE THREAD

**Issue by Hypenate** [date-fns/date-fns#3249]
There is an issue with the [formatISODuration documentation](https://date-fns.org/v2.29.3/docs/formatISODuration)

  ### The problem with formatISODuration documentation
The URL to digi.com is invalid

  ### Suggested edits
Maybe point to https://en.wikipedia.org/wiki/ISO_8601 ?

**Comment by fturmel** [comment]
Agreed, I would deeplink to the durations section specifically: https://en.wikipedia.org/wiki/ISO_8601#Durations

The old link can still be found on the internet archive, but doesn't cover anything that's not already on the wikipedia page.

https://web.archive.org/web/20220324005637/https://www.digi.com/resources/documentation/digidocs/90001437-13/reference/r_iso_8601_duration_format.htm

<img width="1151" alt="Screenshot 2022-10-27 at 10 07 23 AM" src="https://user-images.githubusercontent.com/4009209/198307797-d7c67950-3316-4bdd-8669-28b53428d289.png">
